docs/src/components/Playground/renderConfig.ts

```diff
@@ -17,13 +20,34 @@ export const babelConfig = {
 }
 
 export const imports = {
```

Comment:
```suggestion
export const imports: Record<string, { version: string, module: any }> = {
```

One more thing, can we introduce a type for this?